packages/legacy/core/App/screens/ContactDetails.tsx

```diff
@@ -99,36 +143,86 @@ const ContactDetails: React.FC<ContactDetailsProps> = ({ route }) => {
     [connection, store.preferences.alternateContactNames]
   )
 
+  const contactImage = () => {
+    return (
+      <View style={styles.contactImgContainer}>
+        {contactImageUrl ? (
+          <Image style={styles.contactImg} source={toImageSource(contactImageUrl)} />
+        ) : (
+          <Text
+            style={[
+              TextTheme.bold,
+              {
+                fontSize: 0.5 * contactImageHeight,
+                alignSelf: 'center',
+                color: ColorPallet.grayscale.black,
+              },
+            ]}
+          >
+            {contactLabel.charAt(0).toUpperCase()}
+          </Text>
+        )}
+      </View>
+    )
+  }
+
   return (
     <SafeAreaView style={{ flexGrow: 1 }} edges={['bottom', 'left', 'right']}>
-      <View style={styles.contentContainer}>
-        <Text style={{ ...TextTheme.headingThree }}>{contactLabel}</Text>
-        <Text style={{ ...TextTheme.normal, marginTop: 20 }}>
-          {t('ContactDetails.DateOfConnection', {
-            date: connection?.createdAt ? formatTime(connection.createdAt, { includeHour: true }) : '',
-          })}
-        </Text>
+      <View style={[styles.contentContainer, { flex: 2 }]}>
+        <View style={styles.contactContainer}>
+          {contactImage()}
+          <Text style={styles.contactLabel}>{contactLabel}</Text>
+        </View>
+        {showConnectedTime && (
+          <Text style={{ ...TextTheme.normal, marginTop: 20 }}>
+            {t('ContactDetails.DateOfConnection', {
+              date: connection?.createdAt ? formatTime(connection.createdAt, { includeHour: true }) : '',
+            })}
+          </Text>
+        )}
+        <View style={{ borderTopColor: ColorPallet.grayscale.lightGrey, borderWidth: 1, marginTop: 20 }}></View>
+        <Text style={{ ...TextTheme.headingFour, marginVertical: 16 }}>{t('ContactDetails.Credentials')}</Text>
+        <FlatList
+          ItemSeparatorComponent={() => <View style={{ height: 20 }}></View>}
+          ListEmptyComponent={() => (
+            <Text style={{ ...TextTheme.normal, color: ColorPallet.grayscale.lightGrey }}>
+              {t('ContactDetails.NoCredentials')}
+            </Text>
+          )}
+          data={connectionCredentials}
+          renderItem={({ item }) => (
+            <ContactCredentialListItem
+              credential={item}
+              onPress={() => navigation.navigate(Screens.CredentialDetails, { credential: item })}
+            />
+          )}
+          keyExtractor={(item) => item.id}
+        />
```

Comment:
Could the list of credentials here please be configurable / optional as well?

Reply:
@fc-santos just following up, could this credential list be optional as well? Just in case a Bifold team would prefer not to show these credentials? Request from our UX/UI folks